indexer/lib/fwss-handlers.js

```diff
@@ -43,17 +44,20 @@ export async function handleFWSSDataSetCreated(
         id,
         service_provider_id,
         payer_address,
-        with_cdn
+        with_cdn,
+        with_ipfs_indexing
       )
-      VALUES (?, ?, ?, ?)
-      ON CONFLICT DO NOTHING
+      VALUES (?, ?, ?, ?, ?)
+      ON CONFLICT DO UPDATE SET
+        with_ipfs_indexing = excluded.with_ipfs_indexing
```

Comment:
```suggestion
      ON CONFLICT DO NOTHING
```

This handler is for when a data set was created, not when it was updated. Therefore I see no purpose in an updater

Reply:
As discussed on Zoom, I want to update existing datasets in our DB with IPFS-related metadata. We agreed to use `ON CONFLICT DO NOTHING` and delete the DB before re-running the Goldsky pipeline.